lib/puppet/reports/datadog_reports.rb

```diff
@@ -127,7 +127,7 @@ def process
     Puppet.debug "Sending metrics for #{@msg_host} to Datadog"
     @dog.batch_metrics do
       metrics.each do |metric, data|
-        data.each_value do |val|
+        data.values.each do |val| # rubocop:disable Style/HashEachMethods
```

Comment:
error that requires this:
`2025-03-14T15:12:49.135Z ERROR [qtp2067368747-71] [puppetserver] Puppet Report processor failed: undefined method each_value' for #<Puppet::Util::Metric:0x7317bca6>`